dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/common/DefaultIdentifiableObjectManager.java

```diff
@@ -1206,27 +1204,6 @@ public void evict( Object object )
             .build();
     }
 
-    @Override
-    public boolean isDefault( IdentifiableObject object )
-    {
-        Map<Class<? extends IdentifiableObject>, IdentifiableObject> defaults = getDefaults();
-
-        if ( object == null )
-        {
-            return false;
-        }
-
-        Class<?> realClass = getRealClass( object.getClass() );
-
-        if ( !defaults.containsKey( realClass ) )
-        {
-            return false;
-        }
-
-        IdentifiableObject defaultObject = defaults.get( realClass );
-        return defaultObject != null && defaultObject.getUid().equals( object.getUid() );
-    }
-
```

Comment:
This maybe still useful, not sure if we wanna remove it.

Reply:
I decided to keep it removed. IntelliJ shows that it was not used by any other code. So I think it shouldn't be there. Can be added later when needed (and can be found in git log).